### PR TITLE
Null's Best: Clean up, experiments and organized chaos.

### PR DESCRIPTION
**Pull request by Nullmaruzero**
This time I tried to organize changes better within the commits.
In general I tried to reference more translation keys, so we don't have one simple button or phrase translated a dozen of times.

DrWeb may want to take a look at `PreviewNextPicButton` and `PreviewPreviousPicButton`, he voiced some opinions about those being pulled directly from `next` and `previous`.

### CODE REVIEW

**Comment by Willster419** [comment]
reviewing now.
Please take this time to work with @SigmaTel71 to verify that these changes are requested before I approve this change

**Comment by Willster419** [comment]
for logging:

> [12:45 PM] Nullmaruzero: I did not change Russian translations. I asked him for that but left it as it is. But I saw that next is being pulled (it wasn't me)

Continuing to review changes

File: RelhaxModpack/RelhaxModpack/Utils/Translations.cs [hunk 1]
```diff
@@ -3648,13 +3668,13 @@ public static void LoadTranslations()
...
             //Component: SelectedInstallationNone
-            //
-            English.Add("SelectedInstallationNone", "(none)");
-            German.Add("SelectedInstallationNone", "(nichts)");
-            Polish.Add("SelectedInstallationNone", "(brak)");
-            French.Add("SelectedInstallationNone", "(aucun)");
-            Spanish.Add("SelectedInstallationNone", "(ninguna)");
-            Russian.Add("SelectedInstallationNone", "(не выбрана)");
+            // A text label with the (not selected in this case) path to the selected (active) installation.
+            English.Add("SelectedInstallationNone", "("+English["none"].ToLower()+")");
```
Comment (Willster419) [inline RelhaxModpack/RelhaxModpack/Utils/Translations.cs]: nice! 👍

File: RelhaxModpack/RelhaxModpack/Windows/GameCenterUpdateDownloader.xaml [hunk 2]
```diff
@@ -68,10 +68,10 @@
         <TabItem Header="Download Update Files" IsEnabled="False" x:Name="GcDownloadStep4">
             <Grid Background="#FFE5E5E5" Tag="GcDownloadStep4Gred">
                 <Button x:Name="GcDownloadStep4PreviousButton" HorizontalAlignment="Left" Margin="10,0,0,10" VerticalAlignment="Bottom" Width="75" Click="GcDownloadStep4PreviousButton_Click" Tag="GcDownloadStep4PreviousButton">
-                    <TextBlock x:Name="GcDownloadStep4Previous" Text="Previous" HorizontalAlignment="Stretch" VerticalAlignment="Stretch"/>
+                    <TextBlock x:Name="GcDownloadStep4PreviousText" Text="Previous" HorizontalAlignment="Stretch" VerticalAlignment="Stretch"/>
```
Comment (Willster419) [inline RelhaxModpack/RelhaxModpack/Windows/GameCenterUpdateDownloader.xaml]: Nice find. Consistency is key, but when you are "in the zone", sometimes you forget in your rush to get all the code from your brain to the screen 😄

Comment (Nullmaruzero) [reply RelhaxModpack/RelhaxModpack/Windows/GameCenterUpdateDownloader.xaml]: Thanks. Yea, I guess with all the V1 bits, some oldies, copy-paste leftovers and me coming to the scene pretty late, the best way would it to go top to bottom with such things, but that would take time plus I know that I would feel the urge to move things around within the Translation.cs file. I am the revolution over evolution type, which is not good for collaborations so I will hopefully hunt stuff down slowly and steadily :D

**Pull request merged**